devashish_170102026.py

```diff
@@ -1,33 +1,34 @@
 import numpy as np
 import matplotlib.pyplot as plt
-c='data.txt'
-x=np.loadtxt(c,delimiter='	',skiprows=1)
-#	DELIMITER HERE IS A TAB SPACE
-
-#	FOR DRAWING ALL POSSIBLE PLOTS
- 
-#for j in range (1,11):
-#	for k in range (1,11) :
-#		for i in range (1,999) :
-#			if (x[i,0] == 1) :
-#				plt.scatter(x[i,j],x[i,k],color='g',marker = 'x')
-#			elif (x[i,0] == 2) :
-#				plt.scatter(x[i,j],x[i,k],color='y',marker = 'o')
-#		print(j)
-#		print(k)
-#		plt.show()
+x=np.loadtxt('data.txt',delimiter='\t',skiprows=1)
 
+#	DELIMITER HERE IS A TAB SPACE
 
 for i in range (1,999) :
 	if (x[i,0] == 1) :
 		plt.scatter(x[i,1],x[i,2],color='r',marker = 'x')
-	elif (x[i,0] == 2) :
+	else:
 		plt.scatter(x[i,1],x[i,2],color='b',marker = 'o')
 plt.title('features')
 plt.xlabel('1st attribute')
 plt.ylabel('2nd attribute')
-
 plt.show()
 
+l=0
 
-
+#	FOR DRAWING ALL POSSIBLE PLOTS
+for j in range (1,11):
+	for k in range (1,11) :
+		if (j < k) :
+			for i in range (1,999) :
+				if (x[i,0] == 1) :
+					plt.scatter(x[i,j],x[i,k],color='r',marker = 'x')
```

Comment:
You'll say I'm a bit picky, but since we're down that line, so I might as well say. Once you write large codes, you'll notice that it becomes hard for you to track what parameter values have been passed. So, it's recommended that you add a space after each comma, and write `marker='x'` instead of `marker = 'x'`.

Reply:
I'll take care of this next time onward..